docker-compose.yml

```diff
@@ -16,6 +16,7 @@ services:
       - ./app:/app
       - /app/experimenter/legacy-ui/core/.cache/
       - /app/experimenter/legacy-ui/core/node_modules/
+      - /app/experimenter/reporting/reporting-ui/node_modules/
```

Comment:
Make sure to add this to every `app:dev` container below so they all have the same list of volumes